manual/embedded-examples/src/docs/dev/index.asciidoc

```diff
@@ -4,6 +4,17 @@
 It's easy to use Neo4j embedded in Java applications.
 In this chapter you will find everything needed -- from setting up the environment to doing something useful with your data.
 
+[WARNING]
+====
+When running your own code alongside Neo4j in the same JVM, there are a few things you should keep in mind:
```

Comment:
When running your own code and Neo4j in the same JVM, there are a few things you should keep in mind: